block-storage/src/main/scala/coop/rchain/blockstorage/dag/DagMessageState.scala

```diff
@@ -0,0 +1,188 @@
+package coop.rchain.blockstorage.dag
+
+import cats.syntax.all._
+
+// DagMessageState represents state of one validator in the network
+final case class DagMessageState[M: Ordering, S: Ordering](
```

Comment:
Will there be other states that describes local copy of a DAG? I'm trying to understand why the is not just DagState